stablehlo/dialect/VhloBase.td

```diff
@@ -59,43 +59,6 @@ def VHLO_VersionedTypeInterface : TypeInterface<"VersionedTypeInterface"> {
   ];
 }
```

Comment:
Should `VHLO_VersionedTypeInterface` move to `VhloTypes.td` (and the same thing about other `VHLO_VersionedFooInterface` and the corresponding `VhloFoos.td` files).

Reply:
This seems to work out well, except that we need to have `VhloEnums.td` include `VhloAttrs.td` (and not the other way around, which is how things currently are in the code.).

Reply:
If we decide to go this route the `include "mlir/IR/AttrTypeBase.td"` can be moved from `VhloBase.td` to `VhloTypes.td`.

Reply:
> we need to have VhloEnums.td include VhloAttrs.td

Could you clarify this? Currently Attrs includes Enums, are you suggesting the proper way is to reverse that dependency?

> Should VHLO_VersionedTypeInterface move to VhloTypes.td

Not necessarily. We could organize it that way, but interfaces have no dependencies to (op|attr|type)-defs, so the current organization is "base declares all interfaces, and the more specific files implement the interfaces for definitions."

I'm impartial, let me know what you think.

> If we decide to go this route the include "mlir/IR/AttrTypeBase.td" can be moved from VhloBase.td to VhloTypes.td.

Done.

Reply:
Discussed this offline and decided to move interfaces.